omod/src/main/webapp/resources/scripts/continuityOfCare.js

```diff
@@ -0,0 +1,23 @@
+function isCCDAvailable() {
+    return true;
+}
+
+function getCCDDate() {
+    return "DD/MM/YYYY"
+}
+
+function viewCCD() {
+    alert("Document is being viewed.");
+}
+
+function importCCD() {
```

Comment:
Creating a backend conroller (that does nothing at the moment is also part of this ticket).

Also - does this need to live in registration app? How about XDS sender?

Reply:
You're right. I'll do the controller.

However about the xdssender - I tried to put it here, but it was not visible. It's possible that I did sth wrong, but I've spent a lot of time on it, so I finally left it in the registrationapp.

Reply:
The controller has to be in the same module what fragment specification, so I moved it from isanteplus here. In the end, it might be even better place for it.